plots/01_heliostat_position.py

```diff
@@ -239,11 +239,11 @@ def plot_heliostat_positions(self):
     sys.argv = [
         "heliostat_position.py",
         "--path_to_positions",
-        "data/Heliostatpositionen_xyz.xlsx",
+        "preprocessing/Heliostatpositionen_xyz.xlsx",
         "--path_to_measurements",
-        "data/calib_data.csv",
+        "preprocessing/calib_data.csv",
         "--path_to_deflectometry",
-        "data/deflec_availability.xlsx",
+        "preprocessing/deflec_availability.xlsx",
```

Comment:
Not sure about these paths.

Reply:
These were old debugging strings (I think from Max). I have removed them, since they can be provided when starting the script!